### ISSUE TITLE
Breaking change in 1.11.5 working dir is changed

### ISSUE THREAD

**Issue by Legion2** [beryx/badass-runtime-plugin#74]
Version 1.11.5 changed the generated script templates. Now the working dir is set by these scripts to one relative to the runtime. This makes it impossible for the java program to access the working directory from where it was invoked. 
Also the behavior is different on unix and windows. On windows the base directory of the runtime including the scripts is set and on unix the bin subdir is set as working directory.

**Comment by siordache** [comment]
You're right, this change was a really bad idea. I reverted the script templates in version 1.11.6.
